webhook.sh

```diff
@@ -1,7 +1,4 @@
-#!/usr/bin/env bash
-- docker push $DOCKER_USERNAME/my-test-image:latest
-- docker login --username=$HEROKU_LOGIN --password=$HEROKU_API_KEY registry.heroku.com
-- docker tag $DOCKER_USERNAME/my-test-image registry.heroku.com/devops-project-app/web
-- docker push registry.heroku.com/devops-project-app/web
-- heroku container:release web -a devops-project-app
-- curl -dH -X POST "$WEBHOOK_URL"
\ No newline at end of file
+#!/bin/bash
+docker push $DOCKER_USERNAME/my-test-image:latest
+curl -dH -X POST "$WEBHOOK_URL"
+curl -dH -X POST "$cd_webhook" 
```

Comment:
cd_webhook is not an environment variable. It's an output variable from terraform. You can access it in the middle of a bash command with `$(terraform output -raw cd_webhook)`

Once that works, you can delete the previous curl command (along with deleting any manually created resources that still exist. Everything should now be managed via Terraform)